frontend/src/pages/connectionTypes/ConnectionTypesTableRow.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react';
+import { Td, Tr } from '@patternfly/react-table';
+import {
+  Flex,
+  Icon,
+  Label,
+  Switch,
+  Text,
+  TextContent,
+  Timestamp,
+  TimestampTooltipVariant,
+  Tooltip,
+} from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import { ConnectionTypeConfigMapObj } from '~/concepts/connectionTypes/types';
+import { relativeTime } from '~/utilities/time';
+import { updateConnectionTypeEnabled } from '~/services/connectionTypesService';
+
+type ConnectionTypesTableRowProps = {
+  obj: ConnectionTypeConfigMapObj;
+  onUpdate: () => void;
+};
+
+const ConnectionTypesTableRow: React.FC<ConnectionTypesTableRowProps> = ({ obj, onUpdate }) => {
+  const [statusMessage, setStatusMessage] = React.useState<string | undefined>();
+  const [errorMessage, setErrorMessage] = React.useState<string | undefined>();
+  const pendingEnabledState = React.useRef<'true' | 'false' | undefined>();
+  const createdDate = obj.metadata.creationTimestamp
+    ? new Date(obj.metadata.creationTimestamp)
+    : undefined;
+
+  const onUpdateEnabled = async (enabled: boolean) => {
+    setStatusMessage(enabled ? 'Enabling...' : 'Disabling...');
+    setErrorMessage(undefined);
+    pendingEnabledState.current = enabled ? 'true' : 'false';
+
+    const response = await updateConnectionTypeEnabled(obj, enabled);
+    if (response.success) {
+      onUpdate();
+      return;
+    }
+
+    setStatusMessage('Failed');
+    setErrorMessage(response.error || `Failed to ${enabled ? 'enable' : 'disable'}`);
+  };
+
+  React.useEffect(() => {
+    if (
+      pendingEnabledState.current !== undefined &&
+      obj.metadata.annotations?.['opendatahub.io/enabled'] === pendingEnabledState.current
+    ) {
+      setStatusMessage(undefined);
+      pendingEnabledState.current = undefined;
+    }
+  }, [obj.metadata.annotations]);
+
+  return (
+    <Tr>
+      <Td dataLabel="Name" width={70}>
+        <Flex direction={{ default: 'column' }} gap={{ default: 'gapSm' }}>
+          <TextContent>
+            <Text data-testid="connection-type-name">
+              {obj.metadata.annotations?.['openshift.io/display-name'] || obj.metadata.name}
+            </Text>
+          </TextContent>
+          <TextContent>
+            <Text
+              data-testid="connection-type-description"
+              style={{ color: 'var(--pf-v5-c-content--blockquote--Color)' }}
+            >
+              {obj.metadata.annotations?.['openshift.io/description']}
+            </Text>
+          </TextContent>
+        </Flex>
+      </Td>
```

Comment:
`TableRowTitleDescription` doesn't use a flex and just stacks elements. Can we do the same here with just `div` blocks? The flex and text components seems excessive.

Mocks say to use `--pf-v5-global--Color--200` for the description.